azure-pipelines-ui-tests.dfe.yml

```diff
@@ -224,3 +225,212 @@ jobs:
     inputs:
       path: tests/robot-tests/test-results/
       artifactName: test-results-admin-public
+
+
+- job: 'PublicPlaywrightUItest'
+  displayName: Public suite - Playwright UI tests
+  timeoutInMinutes: 160
+  cancelTimeoutInMinutes: 10
+  condition: succeededOrFailed()
+  pool: ees-ubuntu2204-large
+  workspace:
+    clean: all
+  steps:
+  - checkout: self
+    clean: true
+    fetchDepth: 5
+    fetchTags: false
+
+  - task: NodeTool@0
+    displayName: Use Node version 18 above
+    timeoutInMinutes: 5
+    inputs:
+      versionSpec: "18.18.2"
+
+  - script: |
+        node -v
+    displayName: 'node version'
+
+  - script: |
+      npm ci
+    displayName: 'Install CI'
+    workingDirectory: $(System.DefaultWorkingDirectory)/tests/playwright-tests/
+
+  - script: |
+      npx playwright install --with-deps
+    displayName: 'Install Playwright browsers'
+    workingDirectory: $(System.DefaultWorkingDirectory)/tests/playwright-tests/
+        # runs the playwright tests
+
+  - task: AzureKeyVault@2
+    displayName: 'Azure Key Vault: s101d01-kv-ees-01'
+    inputs:
+      azureSubscription: $(SPN_NAME)
+      KeyVaultName: s101d01-kv-ees-01
+      SecretsFilter: ees-test-SLACK-BOT-TOKEN,ees-test-SLACK-TEST-REPORT-WEBHOOK-URL
+      RunAsPreJob: true
+
+  - script: |
+      npx playwright test --project "public"
+    displayName: "Run Playwright tests"
+    workingDirectory: $(System.DefaultWorkingDirectory)/tests/playwright-tests/
+    env:
+            CI: 'true'
```

Comment:
I think the indentation level here is a bit out of line with the others - looks like it should be brought back to 6 spaces?